packages/apps/composer-app/src/playwright/single-player.spec.ts

```diff
@@ -39,6 +39,30 @@ test.describe('Single-player tests', () => {
     });
   });
 
+  test('create folder', async () => {
+    await host.createSpace();
+    await host.createFolder();
+    await waitForExpect(async () => {
+      expect(await host.getFoldersCount()).to.equal(1);
+    });
+  });
+
+  test('delete folder', async () => {
```

Comment:
make this a describe block with two tests:
- delete a folder with a doc in it
- delete a folder with another folder inside it